lib/kubeclient.rb

```diff
@@ -8,9 +9,36 @@
 require 'kubeclient/replication_controller'
 require 'kubeclient/entity_list'
 require 'kubeclient/kube_exception'
-
+require 'kubeclient/watch'
 
 module Kubeclient
+
+  # HTTP Stream used to watch changes on entities
+  class WatchStream
```

Comment:
afaik it's considered best practice to put one class per file - that is to put WatchStream in a separate file.
But I'd like to hear other opinions as well. @tenderlove ? @Fryguy ?